### ISSUE TITLE
EMVideoView continue black or white seconds when activity resume

### ISSUE THREAD

**Issue by recwert** [brianwernick/ExoMedia#345]
- [ ] I have verified there are no duplicate active or recent bugs, questions, or requests

###### Include the following:
 - ExoMedia version: `3.0.5`
 - Device OS version: `5.1`
 - Devide Manufacturer: `Hisense`
 - Device Name: `Hisense E70-T`
 
###### Reproduction Steps
 1.VideoActivity onCreate:
`emVideoView.setScaleType(ScaleType.NONE);
  emVideoView.setKeepScreenOn(true);`
 2.startActivity to another activity
`Intent intent = new Intent(this, ProjectDetailsActivity.class);
  startActivity(intent);`
 3.press back key back to the VideoActivity,
###### Expected Result
the EMVideoView show video right immediately
###### Actual Result
the EMVideoView continue black or white seconds(audio is normal), after seconds, the EMVideoView show video right

**Comment by brianwernick** [comment]
Even though I have marked this as a `Bug`, I'm not sure if there is much I can do due to the `SurfaceView` and `TextureView` surface functionality and how the ExoPlayer (and other VideoPlayers) interact with them. I'll see if there are any optimizations I can do to minimize the duration of the blank screen (this should just be the color of the activity or other views behind the EMVideoView bleeding through)

**Comment by frel** [comment]
Perhaps this is of interest https://github.com/google/ExoPlayer/issues/852

**Comment by brianwernick** [comment]
Hmm, interesting... For reference, the link ojw28 points to is now invalid (because of repo changes); the full versioned link is https://github.com/google/ExoPlayer/blob/6a5cd6889258df6b9a08b770e2bf351a5bb30e18/demo/src/main/java/com/google/android/exoplayer/demo/PlayerActivity.java#L193

**Comment by brianwernick** [comment]
I'm having a difficult time reproducing the issue in the current `master` branch on KitKat through Nougat. Are there other devices that you have noticed this with?

**Comment by rohan31710** [comment]
I got the same issue. But why my last rendered frame is not visible after resuming the activity

**Comment by brianwernick** [comment]
> You can observe this issue in all devices above 4.3. Even with demo app you will see the same issue.
Nope, I have 8 different devices I have tried this with that don't have the issue as I mentioned above. What I do see is the `TextureView` or `SurfaceView` being blank for a moment however no audio is played until I see video frames. If you can reproduce it I would appreciate if you would look in to the issue and find the culprit

If you are wondering why the last frame isn't still visible it is because of how the `SurfaceView` and `TextureView` handle data.

**Comment by ArtworkAD** [comment]
@brianwernick I see exactly the issue you describe on my Nexus 4 5.1, ExoMedia 4.0.2

**Comment by leimenghao** [comment]
@brianwernick  I get the same black-screen issue on my MI5, android 7.0

**Comment by kaka2009** [comment]
how to solve it？@brianwernick

**Comment by kaka2009** [comment]
how to solve it？@ArtworkAD

**Comment by 1242128273wangpeng** [comment]
Is it still unsolved ？

**Comment by RomanSytnyk** [comment]
I have the same problem.
Android 7.0, Samsung SM-T825.
Audio starts before video (2-3 secs), blank screen and audio.

**Comment by hamzaahmedkhan** [comment]
Same problem

**Comment by lmislm** [comment]
use TextureView can solve this problem.

**Comment by impeterwayne** [comment]
can i set useTextureViewBacking programatically

**Comment by Romie0606** [comment]
Is this issue still unsolved? please help.

**Comment by vantangst** [comment]
> Is this issue still unsolved? please help.

check here: https://github.com/brianwernick/ExoMedia/issues/345#issuecomment-281441526
